### PR TITLE
Example of float label rows with and without formatters 

### PR DESCRIPTION
**Pull request by KarinaFernandez**
- FloatLabelRow and FloatLabelTextField added. Those classes already exist in Eureka.
- In the example project a viewController has been added, where an example of each FloatLabelRow is added and a custom formatter is created. 

**Issue:** 
[Issue 988](https://trello.com/c/n9kcz053/65-https-github-com-xmartlabs-eureka-issues-988-create-a-new-row)

**Image:**
floatrows

### CODE REVIEW

File: Sources/FloatLabelRow.swift [hunk 1]
```diff
@@ -352,68 +370,89 @@ public class ZipCodeFloatLabelCell: _FloatLabelCell<String>, CellType {
     }
 }
 
+// MARK: Rows with formatter
+open class _IntFloatRow: FloatLabelRow<IntFloatLabelCell> {
+    public required init(tag: String?) {
```
Comment (mats-claassen) [inline Sources/FloatLabelRow.swift]: Please add empty line after class declarations and before closing class declaration

File: Example/Example/ViewController.swift [hunk 2]
```diff
@@ -5,52 +5,177 @@
...
+    }
+}
+
+class LogoView: UIView {
+    
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        let imageView = UIImageView(image: UIImage(named: "Xmartlabs"))
+        imageView.frame = CGRect(x: 0, y: 0, width: 318, height: 130)
+        imageView.autoresizingMask = .flexibleWidth
```
Comment (santiagofm) [inline Example/Example/ViewController.swift]: if you have a nib for this view i don't think you really need to layout the `imageview` programmatically and you could just set up constraints using the interface builder

File: Example/Example/ViewController.swift [hunk 3]
```diff
@@ -5,52 +5,177 @@
...
+}
+
+// MARK: Logo view
+class LogoViewNib: UIView {
+    
+    @IBOutlet weak var imageView: UIImageView!
+    
+    required init?(coder aDecoder: NSCoder) {
+        super.init(coder: aDecoder)
+    }
```
Comment (santiagofm) [inline Example/Example/ViewController.swift]: Please add an empty line at the end of the class

File: Example/Example/ViewController.swift [hunk 4]
```diff
@@ -5,52 +5,177 @@
...
+        imageView.frame = CGRect(x: 0, y: 0, width: 318, height: 130)
+        imageView.autoresizingMask = .flexibleWidth
+        self.frame = CGRect(x: 0, y: 0, width: 318, height: 130)
+        imageView.contentMode = .scaleAspectFit
+        addSubview(imageView)
+    }
+    
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
```
Comment (santiagofm) [inline Example/Example/ViewController.swift]: Please add an empty line at the end of the class

File: Sources/DecimalFormatter.swift [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+    
+    override open func getObjectValue(_ obj: AutoreleasingUnsafeMutablePointer<AnyObject?>?, for string: String, range rangep: UnsafeMutablePointer<NSRange>?) throws {
+        guard obj != nil else { return  }
+        let str = string.components(separatedBy: CharacterSet.decimalDigits.inverted).joined(separator: "")
+        obj?.pointee = NSNumber(value: (Double(str) ?? 0.0)/Double(pow(10.0, Double(minimumFractionDigits))))
+    }
+    
+    open func getNewPosition(forPosition position: UITextPosition, inTextInput textInput: UITextInput, oldValue: String?, newValue: String?) -> UITextPosition {
+        return textInput.position(from: position, offset:((newValue?.characters.count ?? 0) - (oldValue?.characters.count ?? 0))) ?? position
+    }
```
Comment (santiagofm) [inline Sources/DecimalFormatter.swift]: Please add an empty line at the end of the class

**Pull request merged**